terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/logging/LoggingTest.java

```diff
@@ -50,7 +50,7 @@ public void test01_01_createDefaultXTrackMDC() {
 
         // cut x-Track MDC
         String targetMdc = driver.findElement(By.id("xTrackMDC")).getText();
-        String footerMdc = driver.findElement(By.cssSelector("p")).getText();
+        String footerMdc = driver.findElement(By.id("xtrack")).getText();

```

Comment:
@btyanagawamg 
Plesae create new issue to consider role of the `WebDriverOperations` .
In this issue, you don't need to fix this comment.

Thanks.